webpack/stories/index.js

```diff
@@ -77,41 +78,29 @@ storiesOf('Charts', module)
   });
 
 storiesOf('Power Status', module)
-  .add('Loading', () => (
-    <PowerStatus
-      loadingStatus="PENDING"
-    />
-  ))
+  .add('Loading', () => simpleLoader('xs'))
   .add('ON', () => (
-    <PowerStatus
-      state="on"
-      title="on"
-      loadingStatus="RESOLVED"
-      statusText="On"
-    />
+    <span
+        className={'fa fa-power-off host-power-status on'}
```

Comment:
hmmm.. this is not using the components at all?? the whole point was to reuse the components not just to show css?